src/components/sw360/AddAdditionalRoles/AddAdditionalRoles.tsx

```diff
@@ -58,7 +80,7 @@ function AddAdditionalRoles({ documentType, setDataInputList, inputList, setInpu
                                     <select
                                         className='form-select'
                                         key=''
-                                        name='role'
+                                        name='key'
```

Comment:
This may effect other components that are using this component

Reply:
@hoangnt2 , as we are using the data type to be InputKeyValue[] enum, so it won't effect what I believe.